### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by Lakshan18** [community/community#65887] [New to GitHub]
how to setup on coding language percentage of my project? it is not show my display

**Comment by Linh777GKOG** [comment]
What programming language or development environment are you using for your project?   
If you're comfortable sharing, please provide a code sample or the specific part of your code that displays the project completion percentage.

**Comment by Linh777GKOG** [comment]
Are you encountering any error messages or issues when attempting to display the percentage? If so, please share them.

**Comment by w8ste** [comment]
@Lakshan18 It is usually displayed automatically after pushing files to your GitHub Repository. You might need to wait a bit until it updates, even though it should not take too long. 

If you are encountering any bugs you can also contact the [GitHub support](https://support.github.com/)
